go/grpcwebproxy/main.go

```diff
@@ -80,6 +81,14 @@ func main() {
 			grpcweb.WithWebsocketPingInterval(*websocketPingInterval),
 		)
 	}
+
+	if len(*flagAllowedHeaders) > 0 {
+		options = append(
+			options,
+			grpcweb.WithAllowedRequestHeaders(append([]string{"*"}, *flagAllowedHeaders...)),
```

Comment:
Should the `"*"` be here? I don't think we should add that by default.

Reply:
OK, I would remove this.